client/webserver/site/src/css/main.scss

```diff
@@ -389,6 +389,7 @@ div.popup-notes {
 
 .errcolor {
```

Comment:
Little odd to have the `word-break` style set in a color-related class.  I noticed there's a bootstrap class called `text-break` for this: https://getbootstrap.com/docs/5.0/utilities/text/#word-break
Probably we should make elements into the bootstrap `text-break` class where desired.  @buck54321 can decide though.

Reply:
Yeah, figured out `.errcolor` is only applied on all errors. instead of adding the bootstrap `text-break` class to elements displaying errors.
> Probably we should make elements into the bootstrap text-break class where desired. I'm inclined to say we should use a bootstrap class where there is one. The bootstrap project can update their css as needed and we don't have to worry about it

This is a good reason to stick with bootstrap.